kitsune/products/migrations/0015_auto_20240722_0449.py

```diff
@@ -0,0 +1,527 @@
+        "old_slug": "thunderbird-controls-and-buttons",
+        "title": "Settings",
+        "slug": "settings",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Calendar",
+        "old_slug": "calendar",
+        "title": "Calendar",
+        "slug": "calendar",
+        "parent_topic_slug": "email-and-messaging",
+    },
+    {
+        "old_title": "Install and update",
+        "old_slug": "install-and-update-2",
+        "title": "Installation and updates",
+        "slug": "installation-and-updates",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Thunderbird versions and languages",
+        "old_slug": "thunderbird-versions-and-languages",
+        "title": "Settings",
+        "slug": "settings",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Copy your personal information from one Thunderbird to another",
+        "old_slug": "copy-your-personal-information-from-one-thunderbird-to-another",
+        "title": "Import and export email",
+        "slug": "import-and-export-email",
+        "parent_topic_slug": "email-and-messaging",
+    },
+    {
+        "old_title": "Emails",
+        "old_slug": "emails",
+        "title": "Email and messaging",
+        "slug": "email-and-messaging",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Set up email",
+        "old_slug": "set-up-email",
+        "title": "Account management",
+        "slug": "account-management",
+        "parent_topic_slug": "accounts",
+    },
+    {
+        "old_title": "Read, send and organize emails",
+        "old_slug": "read-send-and-organize-emails",
+        "title": "Send and receive email",
+        "slug": "send-and-receive-email",
+        "parent_topic_slug": "email-and-messaging",
+    },
+    {
+        "old_title": "News Feeds (RSS), Blogs and Social",
+        "old_slug": "news-feeds-rss-blogs-and-social-thunderbird",
+        "title": "Settings",
+        "slug": "settings",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Contacts",
+        "old_slug": "contacts",
+        "title": "Contacts",
+        "slug": "contacts",
+        "parent_topic_slug": "email-and-messaging",
+    },
+    {
+        "old_title": "Install and update",
+        "old_slug": "install-and-update",
+        "title": "Install",
+        "slug": "install",
+        "parent_topic_slug": "installation-and-updates",
+    },
+    {
+        "old_title": "Accounts & Sign-in",
+        "old_slug": "accounts",
+        "title": "Accounts",
+        "slug": "accounts",
+        "parent_topic_slug": "",
+    },
+    {
+        "old_title": "Billing & Payments",
+        "old_slug": "payments",
+        "title": "Manage billing",
+        "slug": "manage-billing",
+        "parent_topic_slug": "billing-and-subscriptions",
+    },
+    {
+        "old_title": "Connectivity",
+        "old_slug": "performance-and-connectivity",
```

Comment:
This should be `connectivity` instead.
```suggestion
        "old_slug": "connectivity",
```